Feature find form

ashiyaman opened the pull request:
Hi 
I have added find modal. Can you please check and give you inputs

ashiyaman committed:
Basic layout

ashiyaman committed:
Added modal component

DrewDen committed:
added text in for about us section

DrewDen committed:
added new styles for the about us section

DrewDen committed:
added the rest of the code for about us section

DrewDen committed:
adding in the changes I made to the style sheet

DrewDen committed:
add all the html markup for the how it works section

DrewDen committed:
add the changes made in the style page for the how it works section

snrelghgub committed:
marked up new donation form to request donor & donation info where basic info is required & user must tick agreement checkbox before being allowed to submit their donation form as per requirement

snrelghgub committed:
submit button is now disabled by default until agreement checkbox is ticked by user - used jQuery & added new script file for this new functionality

ashiyaman committed:
Added contact form

snrelghgub committed:
changed the writings to Contact us and submit

snrelghgub committed:
Merge pull request #18 from chingu-voyages/feature-contact-form

Added contact form

DrewDen committed:
added a commit on line 29 to explain the move of the div

DrewDen committed:
Merge branch 'develop' into how-it-works

snrelghgub committed:
fixed minor issue with donate modal not spanning all the way to the bottom vertically

DrewDen committed:
cleaned up code to insure that a new design can be done

DrewDen committed:
cleaned up style code to insure a new design

DrewDen committed:
Merge branch 'how-it-works' of https://github.com/chingu-voyages/v9-toucans-team-12 into how-it-works

DrewDen committed:
Merge pull request #22 from chingu-voyages/how-it-works

Merging this branch into had both teammates take a look at it already.

DrewDen committed:
removed a slash line next to the html tag

ashiyaman committed:
Added images for Donated item gallery

snrelghgub committed:
added new image gathered by andrew to our landing page. no styling included but boostrap's classes - rounded mx-auto d-block - could be used to centre & add rounded corners image nicely for landing page

snrelghgub committed:
added ID references to form & two sections of form to make form readily accessible from css/scss file for styling

snrelghgub committed:
increased height of item description's text area as suggested by ashiya by setting new minimum height to textarea element in form

snrelghgub committed:
reconfigured CSS to use calc function to span Donate modal to bottom of viewport on smaller screen sizes properly & avoid text cluterring as pointed out by andrew

snrelghgub committed:
Merge branch 'develop' into dform

snrelghgub committed:
Merge pull request #23 from chingu-voyages/dform

thank you for your feedback andrew & ashiya - I have added more space to the item description's textarea & also reviewed & reconfigured the CSS for the form so that the form still has its scroll-ability while at the same time spanning all the way to the bottom of the viewport to prevent text cluttering especially on smaller screen sizes

snrelghgub committed:
Merge pull request #26 from chingu-voyages/main-image

when we get to the styling stage our page I suggest we use boostrap's classes - rounded mx-auto d-block to centre & add rounded corners to our image nicely for our landing page, I tried it and it looks nice but of course I would never merge this styling preference of mine to the develop branch at this stage. I will make a personal note though in zenhub so I don't forget to remind you of my idea Cheers

DrewDen committed:
updates for the two comments for a more suitable explanation

DrewDen committed:
moved div below the contact form section to ensure styles can be applied to the whole page

DrewDen committed:
removed styles applied to certain classes and used one class to style two sections

DrewDen committed:
removed a space

ashiyaman committed:
Added Isotope plugin

ashiyaman committed:
refractored code

snrelghgub committed:
nested previous CSS code to use SASS for styling of some sections of main html page

snrelghgub committed:
Merge pull request #27 from chingu-voyages/feature-sass

adding SASS to our project

ashiyaman committed:
Refractored code

ashiyaman committed:
Added search functionality

ashiyaman committed:
Changed filter menu color

ashiyaman committed:
Changed filter menu style

ashiyaman committed:
Updated search when all is clicked

ashiyaman committed:
add

ashiyaman committed:
Merge branch 'develop' into feature-find-form

ashiyaman committed:
Moved the css to sass file

ashiyaman committed:
Refractored code

ashiyaman committed:
Refractored code

ashiyaman committed:
Uniform style

ashiyaman committed:
Added description for donated item

snrelghgub commented:
hi ashiya, thanks for your amazing work on this part of our project. I love your original idea where user is able to select by category. I have the following suggestions to make:
-add a new category for "instrument" (because I love instruments lol) & "others" (for things which haven't been categorised & open up a visual opportunity for 'other' things on display) 
-add a hovering effect for when you hover over on an item  
-use email instead of contact number as primary contact, since email is required by default in our donation form 
-add a descriptive item name & item description for each item & not use lorem ipsum text 
-add the same scrolling effect I used in the donation form for when the Find model overflows 
-add a hammer in the list of item too so we can commemorate Donate n Find's beginnings in our project's visual display & add Andrew's email in it too so people can contact him for a hammer :D 
-there is also a problem where the modal is spanning too much to the left & is not centred properly in the viewport 
-rename the close button "Done" to "Close" & style it with our decided button-color 
-rename title to "Find Items" only but definitely bigger h1 for now until we decide on what font-size to use for all modals during styling 

if you want you can fix these few below out of my list & I'll create a new branch off your branch & make a pull request into yours after I make the rest of changes so I can help out:  
-rewrite the close button "Done" to "Close" & style it with our decided button-color 
-there is also a problem where the modal is spanning too much to the left & is not centred properly in the viewport 
-rename title to "Find Items" only but definitely bigger h1 for now until we decide on what font-size to use for all modals during styling

ashiyaman committed:
Changed contact details

ashiyaman committed:
Added scrollable modal content

ashiyaman committed:
Changed title heading

ashiyaman committed:
Added image for music filter tag

ashiyaman committed:
Added items in donated items

ashiyaman committed:
Changed spelling error

ashiyaman committed:
Changed button color

ashiyaman committed:
Changed donated item layout

snrelghgub committed:
added more descriptive text to half of items for our display of list of items

snrelghgub committed:
wrote more descriptive text for the rest of items in our display of list of items of our Find modal

snrelghgub committed:
minor change to text for description of one time

ashiyaman committed:
Added hover effect

ashiyaman committed:
Merge branch 'findformdescript' into feature-find-form

ashiyaman committed:
Centered both modal

ashiyaman commented:
Thanks Taizy for your suggestions. I have made the changes. Thanks for your help as well. 
I have few points to make:
1. We do not have items that will fit in 'other' category. Do you think we need more items.
2. Since both the modal were spanning towards left, I have corrected it in both modal to have uniformity between both
3. I did not understand what you by adding hammer in list of item. Can you please explain more.
4. I removed the violin pic and instead added piano pic, since it did not match the height of other images
I think I have made rest other changes. Hope you are happy with the changes.

ashiyaman committed:
Refractored code

ashiyaman committed:
Refractored code
